server/db.js

```diff
@@ -7,3 +7,8 @@ const connectDB = async () => {
 };
 
 module.exports = connectDB;
+
+require('dotenv').config({ path: '.env' });
+
+// MONGO_URI = 'mongodb+srv://gill:gill@cluster0.myaqn.mongodb.net/myFirstDatabase?retryWrites=true&w=majority'
```

Comment:
Please remove API keys from the code, you can store them within an `.env` file

Reply:
I have worked on every issue and also build an responsive page. Could I merge this now?